packages/vue/src/components/RenderContent.vue

```diff
@@ -167,6 +167,20 @@ export default {
         script.src = editJsSrc;
         script.async = true;
         document.body.appendChild(script);
+        return new Promise((resolve, reject) => {
+          script.addEventListener('load', resolve);
+          script.addEventListener('error', e => reject(e.error));
+        });
+      }
+    },
+    async ensureWcLoadedAndUpdate() {
+      await this.loadWebComponents();
+      const { onBuilderWcLoad } = window;
+      if (onBuilderWcLoad) {
+        onBuilderWcLoad(BuilderWC => {
+          const builder = BuilderWC.builder;
+          builder.canTrack = !window.builderNoTrack;
+        });
```

Comment:
This closely follows the approach in the Angular SDK:

https://github.com/BuilderIO/builder/blob/1d88dd10836fa492704c2d6fce46d09c80ded6fe/packages/angular/src/app/modules/builder/components/builder-component/builder-component.component.ts#L146-L154

I did not reuse the entire logic from Angular since it involves other things that could cause breaking changes (such as API Key assignment)